### PR TITLE
Brand new newsletter and sharing UI

### PR DESCRIPTION
**Pull request by schlosser**

### CODE REVIEW

File: src/components/ShareBlock/ShareBlock.tsx [hunk 1]
```diff
@@ -68,90 +81,110 @@ const ShareBlock = ({
...
+            <Newsletter county={countyName} stateId={stateId} />
+          </NewsletterTextArea>
+        </ShareRowContentArea>
+      </ShareRow>
+      <ShareRow newsletter={false}>
+        <ShareRowContentArea>
+          <SocialTextAreaWrapper>
+            <SocialTextArea>
+              <ShareInstructionHeader>
+                Share data, save lives
```
Comment (goldblatt) [inline src/components/ShareBlock/ShareBlock.tsx:107]: this seems like a lot of div wrappers?

**Comment by goldblatt** [comment]
hmm the embed didn't work in the preview from the home page?

**Comment by goldblatt** [comment]
I think it's a lil confusing that the embed card we show to share and the actual embedded objects/shared data is different?

File: src/components/ShareBlock/ShareBlock.tsx (hunk 1 above)
Comment (schlosser) [reply src/components/ShareBlock/ShareBlock.tsx:107]: Yeah, lots of flexing in alternating... but I don't think any are superfluous

**Comment by schlosser** [comment]
Fixed this to remove embedding on the homepage, per Mike.

Michael is working on updating the actual embed images to match these "mockups" – will create a job to screenshot them.

**Pull request merged**